libsolidity/ast/ASTJsonConverter.cpp

```diff
@@ -252,6 +252,7 @@ bool ASTJsonConverter::visit(ContractDefinition const& _node)
 {
 	setJsonNode(_node, "ContractDefinition", {
 		make_pair("name", _node.name()),
+		make_pair("documentation", _node.documentation() ? Json::Value(*_node.documentation()) : Json::nullValue),
```

Comment:
Can you pull out the documentation part into a separate PR? There is an issue for that and would be nice to close it off separately.